api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/BasicValidationVTwo.java

```diff
@@ -113,7 +119,7 @@ public BasicValidationVTwo(Map<String, List<String>> varsToValidations){
 		                    // incorrectly generate errors.
 		                    if (isEmpty(value)) {
 		                        if (isRequiredField) {
-		                           errors.put(name, REQUIRED_FIELD_EMPTY_MSG);
+		                           errors.put(name, i18n.text("required_field_empty_msg"));
```

Comment:
Should probably use the constant inplace of this String.
https://github.com/vivo-project/Vitro/pull/179/files#diff-c43254e24a1ff63bb1587b9707c50860R31